Dockerfile

```diff
@@ -71,17 +70,18 @@ RUN \
 EXPOSE 32400/tcp 8324/tcp 32469/tcp 1900/udp 32410/udp 32412/udp 32413/udp 32414/udp
 VOLUME /config /transcode
 
-ARG AUTOUPDATE=false
-ARG TAG=beta
+# TAG can be "autoupdate" or an explicit version, like 1.32.4.7195-7c8f9d3b6
+ARG TAG="autoupdate"
```

Comment:
I'm not certain about making the auto update the default behavior.  It is antithetical to docker but then so is the current default of beta so 🤷 .

Reply:
When I made this comment, I didn't realize that your work is effectively supplanting the public and beta tags.  So, assuming that I have that correct, then this change isn't really a change.